app/service/send_notification.py

```diff
@@ -94,7 +94,7 @@ def send_one_off_notification(
     return {'id': str(notification.id)}
 
 
-def get_reply_to_text(
+def _get_reply_to_text(
```

Comment:
I appreciate the rename. Makes it clear that it's local to the file, thank you!

Reply:
You're welcome. :v: